File: Zeze/Transaction/AbortException.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Zeze.Transaction
{
    public sealed class AbortException : Exception
    {
        public AbortException()
        {
        }

        public AbortException(string msg) : base(msg)
        {

        }
    }
}

File: ZezeJava/src/Zeze/Transaction/AbortException.java
package Zeze.Transaction;

public final class AbortException extends Error {
	/**
	 * 
	 */
	private static final long serialVersionUID = 3068088074472148295L;

	public AbortException() {
	}

	public AbortException(String msg) {
		super(msg);

	}
}